x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_nodejs/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "apm_nodejs",
+  "title": "APM: NodeJS",
+  "description": "Detect abnormal traces, anomalous spans, and identify periods of decreased throughput.",
+  "type": "APM data",
+  "logoFile": "logo.json",
+  "defaultIndexPattern": "apm-*",
+  "query": {
+    "bool": {
+      "filter": [
+        {
```

Comment:
Try and condense the filter term into a single line of JSON for readability,

```
"bool": {
  "filter": [
    {"term": {"agent.name": "nodejs"}}
  ]
}
```

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69